docs/boards/Board - FLYCOLORF7.md

```diff
@@ -0,0 +1,107 @@
+# Board - FLYCOLORF7
+
+This board use the STM32F722RET6 microcontroller and have the following features:
+* High-performance and DSP with FPU, ARM Cortex-M7 MCU with 512 Kbytes Flash 
+* 216 MHz CPU,462 DMIPS/2.14 DMIPS/MHz (Dhrystone 2.1) and DSP instructions, Art Accelerator, L1 cache, SDRAM
+DUAL gyro MPU6000 and ICM20689,could choose mpu6000,more stable and smooth.Or choose ICM20689,higher rate(32K/32K).Choose on CLI mode,experience different feature gyro on same board
```

Comment:
@tobeornottobe81: Please update your user facing documentation - 32 kHz gyro loop rate is no longer supported in Betaflight.